config/m4/compiler.m4

```diff
@@ -449,6 +449,27 @@ AS_IF([test "x$enable_gcov" = xyes],
                                        [AC_LANG_SOURCE([[int main(int argc, char** argv){return 0;}]])])],
       [:])
 
+#
```

Comment:
pls use 2 space lines before/after the new code block

Reply:
done